### PR TITLE
Second pass at cloud init pages

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- added correct copy
- added correct partner logos
- added correct images of contributors
- tweaked layout

## QA

Again, this is not ready for a visual QA, just a code sanity check. Please ignore where there are `//TODO` comments are what's below will be refactored before release. 

This version will be pushed onto a `gh-pages` branch so I can step through the build with the design team to finesse.

### CODE REVIEW

File: _sass/_box.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.box {
+  padding: 2rem 2rem 4.5rem;
+  border: 1px solid $alto-grey;
+  border-radius: 3px;
+
+  img {
```
Comment (anthonydillon) [inline _sass/_box.scss]: Can you target a BEM'ed class here please

File: _sass/_base.scss [hunk 2]
```diff
@@ -1,8 +1,37 @@
...
+    }
+  }
+}
+
+// TEMP
+.source-code {
+
+  .box {
+
+    img {
```
Comment (anthonydillon) [inline _sass/_base.scss]: Can you target a BEM'ed class here please

File: _sass/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  padding: 2rem 1rem 1rem;
+  font-size: .75rem;
+
+  &__links {
+
+    list-style: none;
+    padding: 0;
+    margin: 0;
+
+    li {
```
Comment (anthonydillon) [inline _sass/_footer.scss]: Can you target a BEM'ed class here please

File: _sass/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+.footer {
+  padding: 2rem 1rem 1rem;
+  font-size: .75rem;
+
+  &__links {
+
```
Comment (anthonydillon) [inline _sass/_footer.scss]: Remove white space please

File: _sass/_hero.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  &__cta {
+    display: inline-block;
+    margin-top: 1rem;
+  }
+}
+
+.logo {
+
```
Comment (anthonydillon) [inline _sass/_hero.scss]: Remove white space please

File: _sass/_lists.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
+.list--bullets {
+  margin-left: 0;
+
+  @media only screen and (min-width: $breakpoint-medium) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  li {
```
Comment (anthonydillon) [inline _sass/_lists.scss]: Can you target a BEM'ed class please

File: _sass/_lists.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    list-style: none;
+    box-sizing: border-box;
+
+    @media only screen and (min-width: $breakpoint-medium) {
+      width: 50%;
+    }
+
+    &:before {
+      content: '';
+      background-image: url("http://assets.ubuntu.com/sites/ubuntu/latest/u/img/patterns/tick-orange.svg");
```
Comment (anthonydillon) [inline _sass/_lists.scss]: Can you use the new asset server here please: https://assets.ubuntu.com/v1/c4b02d61-tick-orange.svg

File: _sass/_lists.scss [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      background-repeat: no-repeat;
+      display: block;
+      left: 2px;
+      top: 5px;
+      color: $white;
```
Comment (anthonydillon) [inline _sass/_lists.scss]: Are font style required here as the content is blank?

File: _sass/main.scss [hunk 9]
```diff
@@ -6,4 +6,22 @@
...
+'hero',
+'media-unit';
+
+@media only screen and (min-width: $breakpoint-medium) {
+
+  .flex-wrap {
+    display: flex;
+
+    > .four-col {
+      display: flex;
```
Comment (anthonydillon) [inline _sass/main.scss]: Can you explain this? Wont this disturb the grid? Do you mean to nest flex element?

File: index.html [hunk 10]
```diff
@@ -1,168 +1,309 @@
...
-<section class="row">
 
   <!-- Hero banner -->
-  <div class="twelve-col">
+  <div class="hero">
 
-    <span class="logo">
-      Cloud Init
-    </span>
+    <section class="row">
```
Comment (anthonydillon) [inline index.html]: `.hero` and `.row` can be combined into one element as we have `row row-hero`

File: index.html [hunk 11]
```diff
@@ -1,168 +1,309 @@
...
   </div>
-</section>
 
-<!-- Main selling points -->
-<section class="row">
+<div class="wrapper">
 
-  <div class="twelve-col">
+  <!-- Main selling points -->
+  <section class="row">
```
Comment (anthonydillon) [inline index.html]: Do you not want the wrapper inside the row? Same as the hero section above?

File: index.html [hunk 12]
```diff
@@ -1,168 +1,309 @@
...
+  </section>
+
+  <!-- Compatible Platforms -->
+  <section class="row">
+    <div class="twelve-col">
+      <h2>Works with many popular operating systems</h2>
+      <p>While cloud-init started life in Ubuntu, it is now available for most
+        major linux and FreeBSD operating systems. For cloud image providers,
+        then cloud-init handles many of the differences between cloud vendors
+        automatically -- for example, the official Ubuntu cloud images are
```
Comment (anthonydillon) [inline index.html]: Please encode the `--` to `&mash;`

File: index.html [hunk 13]
```diff
@@ -1,168 +1,309 @@
...
+        identical across all public and private clouds.
+      </p>
+    </div>
+
+    <ul class="equal-height no-bullets twelve-col partner-list">
+      <li class="box three-col">
+          <a href="https://launchpad.net/ubuntu/+source/cloud-init">
+              <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 14]
```diff
@@ -1,168 +1,309 @@
...
+                  <img src="https://assets.ubuntu.com/v1/cbb0d58f-ubuntu.png" alt="Ubuntu logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/4d6054f9-logo-cisco.svg" alt="Cisco">
+      <li class="box three-col">
+          <a href="https://wiki.archlinux.org/index.php/Cloud-init">
+              <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 15]
```diff
@@ -1,168 +1,309 @@
...
+                  <img src="https://assets.ubuntu.com/v1/93d81aac-archlinux.png" alt="Arch Linux logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/a298e7ec-logo-hp.svg" alt="HP">
+      <li class="box three-col">
+          <a href="http://cloud.centos.org/centos/7/images/">
+              <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 16]
```diff
@@ -1,168 +1,309 @@
...
+                  <img src="https://assets.ubuntu.com/v1/9a7c8ce5-centos.png" alt="Centos logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/f7db330d-partner-logo-chaosprime.png" alt="ChaosPrime">
+
+      <li class="box three-col last-col">
+          <a href="http://packages.qa.debian.org/c/cloud-init.html">
+          <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 17]
```diff
@@ -1,168 +1,309 @@
...
+              <img src="https://assets.ubuntu.com/v1/eda6b94a-redhat-logo.svg" alt="RedHat logo">
+          </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/3630a703-partner-logo-allwinnertech.png" alt="Allwinner Technology">
+
+      <li class="box three-col">
+          <a href="http://pellaeon.github.io/bsd-cloudinit">
+              <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 18]
```diff
@@ -1,168 +1,309 @@
...
+                  <img src="https://assets.ubuntu.com/v1/6fa4b259-freebsd.png" alt="FreeBSD logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/23835286-partner-logo-rti.png" alt="Real-Time Innovations">
+
+      <li class="box three-col">
+          <a href="https://getfedora.org/en/cloud/download/">
+              <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 19]
```diff
@@ -1,168 +1,309 @@
...
+                  <img src="https://assets.ubuntu.com/v1/0be4c6c4-fedora.png" alt="Fedora logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/e6bc8a26-partner-logo-osrf.png" alt="OSRF">
+
+      <li class="box three-col">
+          <a href="https://packages.gentoo.org/package/app-emulation/cloud-init">
+              <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 20]
```diff
@@ -1,168 +1,309 @@
...
+                  <img src="https://assets.ubuntu.com/v1/cb8e3497-gentoo.png" alt="Gentoo logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/d27eee64-partner-logo-banana-pro.png" alt="LeMaker">
+
+      <li class="box three-col last-col">
+          <a href="https://susestudio.com/">
+              <div>
```
Comment (anthonydillon) [inline index.html]: Is this div required?

File: index.html [hunk 21]
```diff
@@ -1,168 +1,309 @@
...
+  <section class="row source-code">
+    <h2>Source Code</h2>
 
-<!-- Support channels -->
-<section class="row">
+    <div class="flex-wrap">
 
-  <div class="twelve-col">
+      <div class="four-col">
+        <div class="box">
```
Comment (anthonydillon) [inline index.html]: Believe you can move `box` class up to the `four-col element above and remove this div

File: index.html [hunk 22]
```diff
@@ -1,168 +1,309 @@
...
+        <li>Setting the hostname</li>
+        <li>Generating and setting up ssh private keys</li>
+        <li>Setting up ephemeral mount points</li>
+      </ul>
+    </div>
+  </section>
+
+  <!-- Compatible Platforms -->
+  <section class="row">
+    <div class="twelve-col">
```
Comment (anthonydillon) [inline index.html]: As a rule of thumb, we dont allow test to span more then eight-col.

File: index.html [hunk 23]
```diff
@@ -1,168 +1,309 @@
...
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/f7f8709c-logo-joyent.svg" alt="Joyent logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/4f54c31c-Google-Cloud-Platform.png" alt="Google Cloud logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/9aadcd6b-partner-logo-fujitsu.png" alt="Fujitsu logo">
+        </li>
+      </ul>
+      <a href="#_" class="button--primary right">See our other Ubuntu public cloud partners</a>
```
Comment (anthonydillon) [inline index.html]: Odd href here

File: index.html [hunk 24]
```diff
@@ -1,168 +1,309 @@
...
+            </h3>
+            <p>We have an active IRC community - get involved!</p>
+          </div>
+        </div>
+        <div class="four-col last-col">
+          <div class="box">
+            <h3>
+              <a class="external" href="http://cloudinit.readthedocs.org/">Read the docs</a>
+            </h3>
+            <p>Always read the manual - perhaps your question has already be answered?</p>
```
Comment (anthonydillon) [inline index.html]: Please encode all <space>-<space> to `&mdash;`

**Comment by anthonydillon** [comment]
Looks good @barrymcgee, few comments in code (please ask if something doesn't make sense). Difficult to decipher meaning without QA'ing. Could you add steps to run in the README please.

File: index.html (hunk 11 above)
Comment (barrymcgee) [inline index.html]: @anthonydillon - The wrapper is required to give a max-width and a center column to all content other than the hero banner. See screenshot.

screen shot 2016-01-27 at 11 31 12

File: index.html (hunk 21 above)
Comment (barrymcgee) [inline index.html]: @anthonydillon This markup structure is required to keep the three boxes an equal height with Flexbox.

File: index.html (hunk 23 above)
Comment (barrymcgee) [inline index.html]: @anthonydillon I don't have a link to put here - have requested it from Frankie.

File: index.html (hunk 21 above)
Comment (anthonydillon) [inline index.html]: If all you are doing is vertically aligning then you should use vanilla's (http://ubuntudesign.github.io/vanilla-framework/docs/#utils-mixin-vf-equal-height)

**Pull request merged**